src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests.cs

```diff
@@ -1038,6 +1038,34 @@ public async Task EvaluateLocalObjectFromAssemblyNotRelatedButLoaded()
                 var getterProps = await GetProperties(getterId);
                 Assert.Equal(getterProps[0]?["value"]?["value"]?.Value<int>(), 123);
             });
+
+        [Fact]
+        public async Task EvaluateMethodWithDefaultParam() => await CheckInspectLocalsAtBreakpointSite(
+            $"DebuggerTests.DefaultParamMethods", "Evaluate", 2, "Evaluate",
+            $"window.setTimeout(function() {{ invoke_static_method ('[debugger-test] DebuggerTests.DefaultParamMethods:Evaluate'); 1 }})",
+            wait_for_event_fn: async (pause_location) =>
+            {
+                var id = pause_location["callFrames"][0]["callFrameId"].Value<string>();
+                await EvaluateOnCallFrameAndCheck(id,
```

Comment:
Add tests that pass value for the optional params.
And also maybe a test to mix some of these.

Reply:
They were already there, I will change the name for *GetInt32DefaultAndRequiredParam* so that it will be better visible.
```
("test.GetInt32TwoParams(2)", TNumber(5)),
 ("test.GetInt32TwoParams(3, 2)", TNumber(5))
```